spanner/api/Spanner.Samples.Tests/CopyBackupTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using System;
+
+[Collection(nameof(SpannerFixture))]
+public class CopyBackupTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public CopyBackupTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public void CopyBackup()
+    {
+        DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
+
+        CopyBackupSample copyBackupSample = new CopyBackupSample();
+        string source_Project_id = _spannerFixture.ProjectId;
+        string source_Instance_id = _spannerFixture.InstanceId;
+        string source_backupId = _spannerFixture.BackupId;
+        string target_Project_id = _spannerFixture.ProjectId;
+        string target_Instance_id = _spannerFixture.InstanceId;
+        string target_backupId = "test_"+Guid.NewGuid().ToString().Substring(0,16);
+        DateTimeOffset expireTime = DateTimeOffset.UtcNow.AddDays(7);
```

Comment:
So that the backup is deleted quickly
```suggestion
        DateTimeOffset expireTime = DateTimeOffset.UtcNow.AddHours(12);
```